server/app/durablejobs/RecurringJobExecutionTimeResolvers.java

```diff
@@ -82,4 +84,17 @@ public Instant resolveExecutionTime(Clock clock) {
           .toInstant();
     }
   }
+
+  /** Nightly at 3am local time. Used for the MIGRATE_PRIMARY_APPLICANT_INFO job. */
+  public static final class Nightly3Am implements RecurringJobExecutionTimeResolver {
```

Comment:
suggestion (blocking): Sorry I didn't catch this on the first pass. A test for this time resolver should be added here: https://github.com/civiform/civiform/blob/main/server/test/durablejobs/RecurringJobExecutionTimeResolversTest.java